doc/user_guide/user_guide.md

```diff
@@ -43,32 +43,33 @@ Make sure you pick the file with `-dist-` in the name, because that is the insta
 
 ### Configure JDBC Driver for ExaLoader
 
-ExaLoader is responsible for executing the pushdown query on the Databricks and fetching the results. This section describes how to configure the JDBC driver for ExaLoader.
+ExaLoader is responsible for executing the pushdown query on the Databricks and fetching the results. This section describes how to configure the JDBC driver for ExaLoader. You can find more details in the [Exasol Administration documentation](https://docs.exasol.com/db/latest/administration/aws/manage_drivers/add_jdbc_driver.htm).
 
 #### Upload the Databricks JDBC Driver to BucketFS
 
 1. Download the latest [Databricks JDBC Driver](https://www.databricks.com/spark/jdbc-drivers-archive).
 2. Unpack the downloaded ZIP file.
-2. Upload file `DatabricksJDBC42.jar` to BucketFS under path `default/drivers/jdbc/`, see the [BucketFS documentation](https://docs.exasol.com/db/latest/administration/on-premise/bucketfs/accessfiles.htm) for details.
+2. Upload file `DatabricksJDBC42.jar` to the default BucketFS bucket under path `drivers/jdbc/databricks/` (i.e. `/buckets/bfsdefault/default/drivers/jdbc/databricks/`), see the [BucketFS documentation](https://docs.exasol.com/db/latest/administration/on-premise/bucketfs/accessfiles.htm) for details.
```

Comment:
Path `/buckets/bfsdefault/default/drivers/jdbc/databricks/` is only valid within an UDF. If you upload the driver to BucketFS it would be just
"bfsdefault/default/drivers/jdbc/databricks/"